config/wrapper/env.conf

```diff
@@ -14,12 +14,16 @@ name = [('https://github.com/avocado-framework-tests/avocado-misc-tests.git', ''
 
 [deps_centos]
 packages = gcc,python3-devel,python3-setuptools,numactl,policycoreutils-python
+[deps_centos9]
+packages = gcc,python3-devel,python3-setuptools,numactl,policycoreutils-python
```

Comment:
Suggestion : As there is no difference between deps_centos and deps_centos9, let us not create a duplication. Let us have the sections only when there is change in package and names. Like deps_centos9_kvm

Reply:
Thanks for the quick review @harihare 

It doesn't work that way. code is designed in such a way that I cant [deps_abc_kvm] without having [deps_abc]

I think that is the reason we can see [deps_rhel] and [deps_rhel7] having same set of package requirements.

Reply:
Thanks for confirming the same.

Reply:
@bskjois how about when we  run this as avocado_setup.py with `--enable-kvm` as i feel it should honor kvm , if not than it is bug we need to see as don't have system handy can you please check and let us know